manager/src/main/java/com/redhat/service/bridge/manager/vault/VaultService.java

```diff
@@ -0,0 +1,11 @@
+package com.redhat.service.bridge.manager.vault;
+
+import com.redhat.service.bridge.infra.models.EventBridgeSecret;
+
+public interface VaultService {
```

Comment:
🤔  I was thinking at the possibility to use this interface as the SPI to abstract over AWS Secret Manager (and provide a mocked/in-memory secret manager for dev/test).
I noticed this PR is adding localstack for local/dev scenario so it is not strictly necessary but what is the idea for the dev/minikube scenario? Do we plan to add localstack to minikube kustomize script?

Reply:
localstack has been added to the dev env and the minikube overlay